cmd/virt-handler/virt-handler.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+# Configure libvirt username and password
+sed -i s/^authname=/authname=$(cat /etc/sasl-secret/username)/g /etc/libvirt/auth.conf
+sed -i s/^password=/password=$(cat /etc/sasl-secret/password)/g /etc/libvirt/auth.conf
```

Comment:
Like @berrange said in comments above it does not secure to provide the password via command line arguments, but I can just to generate auth.file instead of `COPY` and `sed` commands.